### PR TITLE
refactor: clean up backoffice registrations and appsettings

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs [hunk 1]
```diff
@@ -33,31 +33,10 @@ public static class MigrationsHelper
                 .Execute(() =>
                 {
                     logger?.LogInformation("Running EF Migrations.");
-                    RunInternalSequence(sequenceConnectionString, loggerFactory);
                     RunInternalBackOffice(backOfficeConnectionString, loggerFactory);
                 });
         }
 
-        private static void RunInternalSequence(string connectionString, ILoggerFactory? loggerFactory)
```
Comment (ArneD) [inline src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs:41]: Does this exist in the lambda project?

Comment (jvandaal) [reply src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs:41]: Yes , see https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Api.BackOffice.Handlers.Lambda/Function.cs#L88

Comment (jvandaal) [reply src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs:41]: No, not yet. We'll want to run them at the end of [Function.ConfigureServices()](https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Api.BackOffice.Handlers.Lambda/Function.cs#L108)?

Comment (ArneD) [reply src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs:41]: I would do this in the `ctor` and leave `ConfigureServices` purely for registration

Comment (jvandaal) [reply src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs:41]: Done.

**Review by ArneD — APPROVED** [review]

**Review by pgallik — APPROVED** [review]

**Pull request merged**